README.md

```diff
@@ -14,7 +14,7 @@ Releases can be found on the [GitHub Releases page](https://github.com/dso-toolk
 
 Alpha's should be considered unstable. Beta's will not have any breaking changes.
 
-### NPM
+### NPM registry
 ```
 npm install dso-toolkit
```

Comment:
Voor devs hoeft er idd niets te veranderen, dus inderdaad strikt genomen is dit prima. Toch zie ik tegenwoordig vrijwel altijd en overal OOK de `yarn` installatie instructie. Of vinden we die overbodig (yarners weten dat toch wel...?)